src/main/resources/swagger.api/folio-modules/inventory/instanceIngressEvent.json

```diff
@@ -16,7 +16,8 @@
     },
     "InstanceIngressEventMetadata": {
       "type": "object",
-      "$ref": "../common/eventMetadata.json"
+      "$ref": "../common/eventMetadata.json",
+      "description": "Metadata about Folio event"
```

Comment:
This is Instance ingress event metadata, not a Folio one, right?

Reply:
right. Done.